apps/experiments/views/experiment.py

```diff
@@ -963,10 +989,11 @@ def _record_consent_and_redirect(request, team_slug: str, experiment_session: Ex
 def start_session_from_invite(request, team_slug: str, experiment_id: str, session_id: str):
     experiment = get_object_or_404(Experiment, public_id=experiment_id, team=request.team)
     experiment_session = get_object_or_404(ExperimentSession, experiment=experiment, external_id=session_id)
-    consent = experiment.consent_form
+    default_version = experiment.default_version
+    consent = default_version.consent_form
 
     initial = {
-        "experiment_id": experiment.id,
+        "experiment_id": default_version.id,
```

Comment:
This doesn't seem to be used by the form. It's unclear to me if it's needed at all and if so whether it should be the working or default version

Reply:
You're right, it's not being used. Ripping it out...